src/components/NewTableComponent/JobModal.jsx

```diff
@@ -56,8 +57,7 @@ export default function ViewJobModal({ open, setOpen, id }) {
         const fetchJobData = async () => {
             setLoading(true);
             try {
-                const response = await fetch(`${baseUrl}/api/v1/jobs/${id}`);
-                const data = await response.json();
+                const data = await fetchEachJob(id);
```

Comment:
```suggestion
                const data = await fetchJobById(id);
```
update in api.ts file also